src/Base/Chord/Sus.hs

```diff
@@ -1,15 +1,38 @@
 module Base.Chord.Sus
   ( Sus
   , getMaybeDeg
+  , isSus
   , sus
   , noSus
+  , susNoNum
   ) where
 
-newtype Sus = Sus { getMaybeDeg :: Maybe Int }
-  deriving Show
+data Sus
+  = Sus Int
+  | SusNoNum
+  | NoSus
+  deriving(Eq)
```

Comment:
I'm not sure I like this; why can't we stick with `newtype Sus = Sus { getMaybeDeg :: Maybe Int }`? Does `SusNoNum` ever mean anything other than `sus2`? It wasn't obvious to me from the other code that uses this whether that's the case or not.

Reply:
You could have a 13sus chord, which means you're adding the 2 AND 4 as a "sus." The default to 2 misses that case, which is mostly relevant to making the notesToChord function work. I'm not sure if I changed the parser to reflect this but similarly, if you say C7sus, it's usually understood to mean [C, D, F, G, Bb], i.e. 2 and 4 sussed.

Reply:
Ahhh, OK, that makes sense. Unfortunate that it needs to be this way, but that's the price we pay to have nice things.